snews_cs/snews_coinc.py

```diff
@@ -316,12 +312,11 @@ def __init__(self, env_path=None, use_local_db=True, drop_db=False, firedrill_mo
         self.topic_type = "CoincidenceTier"
         self.coinc_threshold = float(os.getenv('COINCIDENCE_THRESHOLD'))
         self.cache_expiration = 86400
-
+        self.pub_state = None
```

Comment:
I'm looking at the code, going to try it soon. But, does this `self.pub_state` is used anywhere else?